src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -10088,8 +10088,6 @@ class Program
 {
     static R F1(ref R r) { Console.WriteLine(r._i); return new R(); }
     static R F2(scoped ref R r) { Console.WriteLine(r._i); return new R(); }
-    static R F3(ref scoped R r) { Console.WriteLine(r._i); return new R(); }
-    static R F4(scoped ref R r) { Console.WriteLine(r._i); return new R(); }
```

Comment:
>static R F4(scoped ref R r) { Console.WriteLine(r._i); return new R(); } [](http://example.com/codeflow?start=4&length=72)

Consider keeping this case and `d4` below, to verify the synthesized delegate is shared between `d2` and `d4`.